core/src/main/java/org/elasticsearch/cluster/ClusterChangedEvent.java

```diff
@@ -152,46 +168,35 @@ public boolean indexMetaDataChanged(IndexMetaData current) {
         return true;
     }
 
-    public boolean blocksChanged() {
```

Comment:
Please don't remove this method here, it would make this a breaking change in that it would break any plugins that relied on watching for ClusterChangeEvents where the blocks have changed